File: app/src/main/java/view/pager/recycler/viewholder/TextItemHolder.java
package view.pager.recycler.viewholder;

import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.TextView;

import view.pager.recycler.R;

/**
 * Created by harneev on 20/09/16.
 */

public class TextItemHolder extends RecyclerView.ViewHolder {

    public TextView tvTitle;

    public TextItemHolder(View view) {
        super(view);

        tvTitle = (TextView) view.findViewById(R.id.tv_item);
    }
}

File: app/src/main/java/view/pager/recycler/view/viewholder/TextItemHolder.kt
package view.pager.recycler.view.viewholder

import android.view.View
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView

import view.pager.recycler.R


class TextItemHolder(view: View) : RecyclerView.ViewHolder(view) {

    var tvTitle: TextView

    init {

        tvTitle = view.findViewById<View>(R.id.tv_item) as TextView
    }
}
